torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -6,13 +6,26 @@
 from .match_utils import MatchAllNode
 from .utils import _parent_name
 
-def _lower_ref_linear_module(model: QuantizedGraphModule) -> QuantizedGraphModule:
-    # traverse the graph and find dequantize - ref quantized linear - quantize patterns and
-    # and replace it with quantized linear modules
-    pattern = (
-        torch.quantize_per_tensor,
-        (torch.nn.quantized._reference.Linear, "dequantize"),
-        MatchAllNode, MatchAllNode, MatchAllNode)
+# Mapping from reference module class to the replacement quantized module class for lowering
+_lower_module_map = {
```

Comment:
nit: we typically use something like `LOWER_MODULE_MAP` to indicate this is a constant map